packages/css/src/components/tabs/tabs.template.ts

```diff
@@ -0,0 +1,39 @@
+import { Tabs } from '@dso-toolkit/sources';
+
+import { html } from 'lit-html';
+import { ifDefined } from 'lit-html/directives/if-defined.js';
+
+export function tabsTemplate({ tabs }: Tabs) {
```

Comment:
Dit template is teveel ingekleurd en lijkt daarom meer op een voorbeeldpagina/component. Het template is daarom niet meer te herbruiken. Er moet meer naar de content en/of modelering.